lib/cartopy/mpl/geoaxes.py

```diff
@@ -412,16 +412,9 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
 
         Kwargs:
 
-            * resolution - a named resolution to use from the Natural Earth
-                           dataset. Currently can be one of "110m", "50m", and
-                           "10m".
-
-        .. note::
```

Comment:
Where did this note go?

Reply:
This was about something which was supposed to be fixed in v0.5, so it's either been fixed (which we're pretty sure it has, having spoken to @pp-mo about it), or it's just not worth pointing out any more.